### PR TITLE
Add Remove Appx packages artifact

### PR DESCRIPTION
**Pull request by azabbasi**

### CODE REVIEW

File: Artifacts/windows-appxremove/remove-appx-packages.ps1 [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    # a non-zero exit code from the catch block.
+    exit -1
+}
+
+###################################################################################################
+#
+# Main execution block.
+#
+
+try
```
Comment (leovms) [inline Artifacts/windows-appxremove/remove-appx-packages.ps1]: `try` is not required, unless you have a `catch` or `finally` block. The `trap` will handle the error for you.

File: Artifacts/windows-appxremove/remove-appx-packages.ps1 [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+{
+    if ([Environment]::OSVersion.Version.Major -gt 6) {
+        Write-Host "`nRemoving Appx Packages`n"
+        Get-AppxPackage -AllUsers | Remove-AppxPackage -ErrorAction SilentlyContinue
+        Write-Host "`nDone Removing Appx Packages`n"
```
Comment (leovms) [inline Artifacts/windows-appxremove/remove-appx-packages.ps1]: Not sure if you need to use \`n for in all your strings. You will end up with blank lines between each. If that's on purpose, ignore my comment.

File: Artifacts/windows-appxremove/remove-appx-packages.ps1 [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+###################################################################################################
+#
+# Main execution block.
+#
+
+try
+{
+    if ([Environment]::OSVersion.Version.Major -gt 6) {
+        Write-Host "`nRemoving Appx Packages`n"
+        Get-AppxPackage -AllUsers | Remove-AppxPackage -ErrorAction SilentlyContinue
```
Comment (leovms) [inline Artifacts/windows-appxremove/remove-appx-packages.ps1]: If you call `Remove-AppxPackage` with `-ErrorAction SilentlyContinue`, then you are potentially never going to fail.

File: Artifacts/windows-appxremove/remove-appx-packages.ps1 (hunk 1 above)
Comment (azabbasi) [reply Artifacts/windows-appxremove/remove-appx-packages.ps1]: done

File: Artifacts/windows-appxremove/remove-appx-packages.ps1 (hunk 3 above)
Comment (azabbasi) [reply Artifacts/windows-appxremove/remove-appx-packages.ps1]: done

**Pull request merged**